app/third_party/github.rb

```diff
@@ -0,0 +1,11 @@
+module Github
+  BASE_URL = "https://api.github.com"
```

Comment:
Create a class to manage this and defined methods to ask base on the request like:
`class GithubService

end
`

Reply:
Maybe you want to parse the response of the body